config/provisioners/kafka/clusterprovisioner.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 The Knative Authors
```

Comment:
I would just smash these all into one yaml file called kafka.yaml, no need to have them broken out like this.